sklego/naive_bayes.py

```diff
@@ -264,6 +265,7 @@ def fit(self, X, y) -> "BayesianGaussianMixtureNB":
                 ).fit(subset_x[:, i].reshape(-1, 1), subset_y)
                 for i in range(X.shape[1])
             ]
+        self.n_iter_ = sum(sum(gmm.n_iter_ for gmm in gmm_c) for gmm_c in self.gmms_.values())
```

Comment:
Isn't `np.sum` a bit faster here?

Reply:
Honestly for such small numbers (`n_classes` x `n_features`) it's not worth it